src/background_updates/run/v1.rs

```diff
@@ -0,0 +1,50 @@
+//! [POST /_synapse/admin/v1/background_updates/start_job](https://github.com/element-hq/synapse/blob/master/docs/usage/administration/admin_api/background_updates.md#run)
+
+use ruma::{
+    api::{request, response, Metadata},
+    metadata,
+};
+
+const METADATA: Metadata = metadata! {
+    method: POST,
+    rate_limited: false,
+    authentication: AccessToken,
+    history: {
+        unstable => "/_synapse/admin/v1/background_updates/start_job",
+    }
+};
+
+#[request]
+pub struct Request {
+    /// A string which job to run
+    ///
+    /// Valid values are:
+    /// populate_stats_process_rooms
+    /// regenerate_directory
+    pub job_name: String,
```

Comment:
question: wondering if we should struct the valid options here. but could be an over kill.

Reply:
Seems reasonable, just define an enum with `#[serde(rename_all = "snake_case")]`. Ruma also has a derive `StringEnum` if you want to allow extra custom values while still having enum variants for the well-known strings (see other uses of that derive macro).